frontends/web/src/api/account.ts

```diff
@@ -408,6 +408,7 @@ export type TUTXO = {
   scriptType: ScriptType;
   addressReused: boolean;
   isChange: boolean;
+  headerTimestamp: string;
```

Comment:
I think in theory both the HeaderTimestamp and the CreatedTimestamp could be `nil`, so that means headerTimestamp should be `string | null`.